ban/tests/test_identifier_redirect.py

```diff
@@ -36,7 +36,7 @@ def test_follow_returns_new_value():
     municipality.increment_version()
     municipality.save()
     assert IdentifierRedirect.select().count() == 1
-    assert IdentifierRedirect.follow(models.Municipality, 'insee', '12345') == '54321'  # noqa
+    assert IdentifierRedirect.follow('Municipality', 'insee', '12345') == ('insee', '54321')  # noqa
```

Comment:
Why having a `# noqa` instead of breaking the line with parenthesis?
Here and below